frontend/src/components/VotePage.tsx

```diff
@@ -5,37 +5,46 @@ import useFetch from "../useFetch";
 import { useParams } from "react-router";
 import React from 'react'
 import { useNavigate } from "react-router";
-
+import { Ballot } from "../../../domain_model/Ballot";
+import { Vote } from "../../../domain_model/Vote";
+import { Score } from "../../../domain_model/Score";
 const VotePage = ({ }) => {
     const {id} = useParams();
     const {data: election, isPending, error} = useFetch(`/API/Election/${id}`)
     const [rankings, setRankings] = useState([])
     const navigate = useNavigate();
-
+    console.log(election)
     const onUpdate = (rankings) => {
         setRankings(rankings)
         console.log(rankings)}
     const submit = () => {
         console.log(election)
         console.log(election.Candidates)
         console.log(rankings)
-        const message = {
-            ElectionID: id,
-            candidateScores: election.Candidates.map((candidate,i) => 
-              ({'id': election.Candidates[i].id, 'score':rankings[i]})
-            )
-          }
-          console.log(message)
+        // const message = {
+        //     ElectionID: id,
+        //     candidateScores: election.polls[0].candidates.map((candidate,i) => 
+        //       ({'id': election.polls[0].candidates[i].candidateId, 'score':rankings[i]})
+        //     )
+        //   }
+        const ballot = {
```

Comment:
nit

This might be another case where ``const ballot: Ballot = {`` would be preferred over ``as Ballot``